app/scripts/store/dimStoreItem.directive.js

```diff
@@ -67,11 +68,26 @@
           ngDialog.closeAll();
 
           if (!dimLoadoutService.dialogOpen) {
-            var bottom = ($(element).offset().top < 300) ? ' move-popup-bottom' : '';
-            var right = ((($('body').width() - $(element).offset().left - 320) < 0) ? ' move-popup-right' : '');
+
+            var settings = dimSettingsService.getSettings().$$state.value,
```

Comment:
This is not guaranteed.  This is why we use the Promise Async syntax.  I'm fine if you really need to peek into the promise to see the $$state but you need to check the state to see if it resolved before you tap in to the value, and write code to handle pending (and rejected) states too.